src/SSVOpenHexagon/Core/Discord.cpp

```diff
@@ -89,32 +89,83 @@ bool discord_manager::set_rich_presence_in_menu()
     }
 
     discord::Activity activity{};
-    activity.SetState("In menu");
-    activity.SetDetails("...");
+    activity.SetState("Selecting Level");
+    activity.SetDetails("");
+    discord::ActivityTimestamps& currentTimestamp = activity.GetTimestamps();
+    currentTimestamp.SetStart(static_cast<int64_t>(
+        std::chrono::high_resolution_clock::now().time_since_epoch().count() /
+        1000000000));

```

Comment:
```suggestion
    currentTimestamp.SetStart(static_cast<std::int64_t>(
        std::chrono::high_resolution_clock::now().time_since_epoch().count() /
        1'000'000'000));

```

Also please include `<cstdint>` here as well.